app/src/main/java/org/fossasia/openevent/dbutils/DbContract.java

```diff
@@ -119,9 +120,9 @@ public DbContract() {
         };
 
         public static final String CREATE_TABLE =
-                "CREATE TABLE " + TABLE_NAME
-                        + " ("
-                        + _ID + " INTEGER PRIMARY KEY,"
+                "CREATE TABLE IF NOT EXISTS " + TABLE_NAME
+                        + " (_id INTEGER PRIMARY KEY,"
+                        // + "_id INTEGER PRIMARY KEY" + COMMA_SEP
```

Comment:
`_id INTEGER PRIMARY KEY" + COMMA_SEP` should be the correct way to go, don't hard code the COMMA_SEP